src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment: { name, email, body } = [] }) => (
```

Comment:
A little bit strange that `comment` is empty array by default)

Looks like your component will crash if accept it (`[]`)